lib/aliases.zsh

```diff
@@ -1,35 +0,0 @@
-# Push and pop directories on directory stack
-alias pu='pushd'
-alias po='popd'
-
-# Basic directory operations
-alias ...='cd ../..'
-alias -- -='cd -'
```

Comment:
Hi @soimort, I've added back the alias because another user was using that as well: https://github.com/robbyrussell/oh-my-zsh/commit/4fbfb149d3edcfe18b7b008c3a4a8e7523d4b811